recipe/install_psycopg2.bat

```diff
@@ -1,4 +1,9 @@
 
+if "%PKG_NAME%" == "psycopg2-binary" (
+  echo renaming psycopg2 to psycopg2-binary
+  "%PYTHON%" %RECIPE_DIR%/rename_psycopg2_binary.py
+)
+
```

Comment:
I'm not super confident in my batch coding skills.  Another pair of eyes would be most welcome!

Reply:
Seems to be working as near as I can tell.